politeiawww/backend.go

```diff
@@ -2045,11 +2035,9 @@ func (b *backend) ProcessComment(c www.NewComment, user *database.User) (*www.Ne
 	}
 
 	// get proposal record from inventory
-	b.RLock()
-	ir, ok := b.inventory[c.Token]
-	b.RUnlock()
-	if !ok {
-		return nil, fmt.Errorf("ProcessComment: inventory proposal not found: %v", c.Token)
+	ir, err := b.getInventoryRecord(c.Token)
+	if err != nil {
+		return nil, fmt.Errorf("ProcessComment: getInventoryRecord: %v %v", c.Token, err)
```

Comment:
"ProcessComment" in the error message is redundant.  If you look at the route handler functions in www.go, they already include "ProcessComment" in the error message. 

https://github.com/decred/politeia/blob/master/politeiawww/www.go#L812

 Same for your other error messages.